src/store/selectors/getPostById.js

```diff
@@ -0,0 +1,13 @@
+import { createSelector as ormCreateSelector } from 'redux-orm'
+import orm from 'store/models'
+
+// protest selector
```

Comment:
protest?

Reply:
hahah, I was annoyed at the time that getPost wouldn't accept you passing in an postId directly. This will all get refractored with the urql changes sooooooon